src/test/java/se/sundsvall/educationfinder/api/CoursesResourceTest.java

```diff
@@ -162,14 +162,14 @@ void findAllOnSpecificStudyLocationAndName() {
 	}
 
 	@Test
-	void findAllOnSpecificStudyLocationAndNameAndEndLessThan() {
+	void findAllOnSpecificStudyLocationAndNameAndEndBefore() {
```

Comment:
Might want to test passing an actual parameter object instead of individual parameters.

Reply:
I'm not sure that I understand?

What parameter object are you referring to?

Reply:
I was mistaken, misread the controller

Reply:
Yes, it is a little bit different since we are using the "specification-arg-resolver" in this service.